__mocks__/react-native-ble-manager.ts

```
@@ -0,0 +1,123 @@
+// Mock the NativeEventEmitter
+const mockEmitter = new NativeEventEmitter();
+const emitEvent = (eventName: string, data: any) => {
+  const mockListener = mockEmitter.addListener.mock.calls.find(
+    call => Array.isArray(call) && call.length > 0 && call[0] === eventName,
+  );
+  if (
+    mockListener &&
+    mockListener.length > 1 &&
+    typeof mockListener[1] === 'function'
+  ) {
+    (mockListener[1] as Function)(data);
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)